Update application.md (#19)
Monofony/Docs: docs/setup/application.md

composer require monofony/api-pack "^0.9"
```

> If you happen to see exception as below, run the migration commands.

Reviewer: ```suggestion
If you see an exception as shown below, run the migration commands.
```

> Class "App\Entity\RefreshToken" sub class of "Gesdinet\JWTRefreshTokenBundle\Entity\RefreshToken" is not a valid entity or mapped super class.

> ```

Reviewer: ```suggestion
```bash
```
